translations/ui-myprofile/pt_BR.json

```diff
@@ -12,5 +12,6 @@
     "settings.changePassword.confirmPasswordMatchError": "Senha nova e a confirmação não correspondem. Digite novamente sua senha.",
     "settings.changePassword.wrongPassword": "Senha incorreta. Digite novamente sua senha.",
     "settings.changePassword.successfullyChanged": "A <strong> senha do FOLIO </strong> foi <strong> alterada </strong> com êxito para <strong> {lastName} </strong> , <strong> {firstName} </strong>",
+    "settings.changePassword.lastTenPasswords": "Previously used password. Please enter a new password.",
```

Comment:
Please remove translations for locales apart `en.json`.

Reply:
Done.